src/manifest_store.rs

```diff
@@ -137,6 +137,10 @@ impl StoredManifest {
         }))
     }
 
+    pub(crate) async fn load(store: Arc<ManifestStore>) -> Result<Self, SlateDBError> {
```

Comment:
nit: let's add the docs above and document the difference between this and `try_load` (since the other one already has docs)